### ISSUE TITLE
tvOS cannot save config

### ISSUE THREAD

**Issue by socksprox** [kodi-pvr/pvr.iptvsimple#817]
After changing something like adding m3u source, after a restart of kodi, it will be gone.

Users already posted on the forum, but maybe here is a better place for developers to notice. 

Forum post: https://forum.kodi.tv/showthread.php?tid=373609

**Comment by phunkyfish** [comment]
I don’t think this can be fixed in the add-on. I think it needs to be fixed in Kodi itself.
